public/app/percona/shared/components/PerconaBootstrapper/PerconaNewVersion/PerconaNewVersion.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { Modal, useStyles2, Button, LinkButton } from '@grafana/ui';
+
+import { MIGRATION_LINK_URL, DOCUMENTATION_LINK_URL } from './PerconaNewVersion.constants';
+import { Messages } from './PerconaNewVersion.messages';
+import { getStyles } from './PerconaNewVersion.styles';
+import { PerconaNewVersionProps } from './PerconaNewVersion.types';
+
+const PerconaNewVersion = ({ isOpen, onDismiss }: PerconaNewVersionProps) => {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <Modal onDismiss={onDismiss} title={Messages.titleNewVersion} isOpen={isOpen} className={styles.newVersionModal}>
+      <div data-testid="new-version-modal">
+        <span>
+          {Messages.description}
+          <a className={styles.link} href={DOCUMENTATION_LINK_URL}>
```

Comment:
I've missed it previously but I think the "see what's new" and "view migration guide" links should open in a new browser tab.